specs/test/testplan.md

```diff
@@ -168,16 +199,55 @@ It should cover the basic function to pass, to fail, and error conditions.
 ---
 
 ## stress test
+
 ### protocol level stress
+
 * test case # : STP1
 * description : 
-* test procedure :
-* passing criteria
+* test procedure : increase number of txns in block from 1000 to 10,000, stepwise
+* evaluation: change in block latency per shard/change in transactions per sec, per shard.
+* dependency
+* note
+* automated?
+---
+* test case # : STP2
+* description : increasing number of nodes in a shard
+* test procedure : increase number of nodes in a shard from 100 to 5000
```

Comment:
don't think we can have one shard with 5000 nodes. Maybe up to 1000?

Reply:
Done.